src/smsCommand/SaveCommandButton.js

```diff
@@ -9,17 +10,35 @@ const subscription = {
     pristine: true,
 }
 
-export const SaveCommandButton = () => {
-    const { submitting, pristine } = useFormState({ subscription })
+export const SaveCommandButton = ({ enabled }) => {
+    const { submitting, pristine } = useFormState({
+        subscription,
+    })
 
     return (
         <Button
             primary
             type="submit"
-            disabled={pristine || submitting}
+            enabled={enabled || pristine || submitting}
             icon={submitting ? <CircularLoader small /> : null}
         >
             {i18n.t('Save command')}
         </Button>
     )
 }
+
+SaveCommandButton.defaultProps = {
+    enabled: false,
+}
+
+SaveCommandButton.propTypes = {
+    /*
+     * This prop is used when fields, stored as array entries
+     * in the form state, are being removed while the form
+     * is still pristine.
+     *
+     * Form some reasons, RFF doesn't recognize this as a change
+     * and keeps the form state's pristine value `true`
+     */
```

Comment:
I'm thinking that maybe we should report this as a bug, to take a closer look later.

Reply:
Yeah, once I'm done with all the PRs I'm reviewing/working on, I'll create an example repo to reproduce this and file a bug in the RFF repo

Reply:
Sounds good, thanks